src/glmfit.jl

```diff
@@ -21,6 +21,8 @@ struct GlmResp{V<:FPVector,D<:UnivariateDistribution,L<:Link} <: ModResp
     wrkwt::V
     "`wrkresid`: working residuals for IRLS"
     wrkresid::V
+    "`link`: to get the link function with relevant parameters"
```

Comment:
```suggestion
    "`link`: link function with relevant parameters"
```

Reply:
Can you also move the field right after `d`? That sounds more logical.

Reply:
Sure

Reply:
As per your suggestions, 
1. For all real number comparisons, used ≈ operator
2. Used approx function wherever we need to provide a tolerance value
3. Added a few more metrics like `stderror`, `disperson`, `confidence interval` in the test cases.